cmd/tf-migrate/main.go

```diff
@@ -330,22 +335,23 @@ func findTerraformFiles(dir string) ([]string, error) {
 }
 
 func validateVersions(c config) error {
-	source := strings.TrimPrefix(c.sourceVersion, "v")
-	target := strings.TrimPrefix(c.targetVersion, "v")
-	versionPath := fmt.Sprintf("%s-%s", source, target)
+	versionPath := fmt.Sprintf("%s-%s", c.sourceVersion, c.targetVersion)
 	if _, ok := validVersionPath[versionPath]; !ok {
 		return fmt.Errorf("unsupported migration path: %s", versionPath)
 	}
-
 	return nil
 }
 
 func getProviders(resources ...string) transform.Provider {
 	getFunc := func(resourceType string, source string, target string) transform.ResourceTransformer {
-		return internal.GetMigrator(resourceType, source, target)
+		sourceVersion, _ := strconv.Atoi(source)
```

Comment:
We're potentially throwing out errors here. Does this get easier and safer if we change the types to `uint`s?